README.md

```diff
@@ -331,7 +346,53 @@ The `getLabel` function in this mode will have the following signature:
 -   `reply`
 -   **Returns:** <`string`> The route label string without any `.` at the beginning or end.
 
-The `this` context of the function is bound to the fastify instance of the request.
+The `this` context of the function is bound to the fastify instance of the request. Pay attention to avoid returing empty strings or strings with leading and trailing `.`. Also, don't use arrow functions otherwhise the `this` context won't refer to the fastify instance.
+
+The default function used ifn you don't pass your custom one returns the same string that is computed in `static` mode, so the `dynamic` mode is not very useful if you don't define yuor own `getLabel` function.
```

Comment:
`ifn` -> `if`; 
 I don't know, but it seems to miss some comma in this sentence.

Reply:
WDYT to rephrase in something like this:
If you don't pass your custom one, the default function returns the exact string computed in `static` mode ...

Reply:
Yes that sounds better, thanks